ui-components/src/locales/general.json

```diff
@@ -831,6 +831,8 @@
   "t.accessibility": "Accessibility",
   "t.additionalAccessibility": "Additional Accessibility",
   "t.additionalPhone": "Additional Phone",
+  "t.addItem": "Add item",
```

Comment:
Once these strings are moved out of the component, we can also move them into the partners overrides strings file since they won't be used on the public side.